src/components/SearchInput.js

```diff
@@ -121,10 +158,12 @@ const SearchInput = (props) => {
               </IconButton>
               <Tooltip
                 classes={{
+                  arrow: classes.arrow,
                   tooltip: classes.tooltip,
                 }}
                 placement="top"
                 open={tooltipOpen}
+                arrow
                 title={
                   <span>
                     Great! You can always switch your search engine here later
```

Comment:
This text feels too small. Maybe use body2? Also, it should appear over the TFAC logo.
![Screen Shot 2022-04-12 at 12 02 12 PM](https://user-images.githubusercontent.com/5924325/163009100-e06d0fb8-c6d2-42b1-8dc5-0b0827a41965.png)